integration-tests/bats/remotes.bats

```diff
@@ -2075,7 +2075,8 @@ SQL
     [ "$status" -eq 0 ]
     [[ "$output" =~ "Everything up-to-date." ]] || false
 
-    run dolt branch --track direct feature origin/other
+    # TODO: this command fails on GitHub requiring `--track=direct`, when both branch name and starting point name are defined, but Dolt allows both formats.
```

Comment:
(minor) You might change this to `NOTE:` instead of `TODO:` – I think this is such a small difference that there's no action we need to take, so it didn't seem like a `TODO` to me. 

When I see "GitHub", I think about the GitHub website, but I think here you actually just mean "with git"?